Bottom: 78ab9ab54e3365ff914a7eb9e65779713ac01532
Top:    505a3ef5484e44a38b74f7c024b2909f08014910
Author: scott snyder <sss@karma>
Date:   2019-12-27 00:47:12 -0500

cad fix..cad fix....


---

diff --git a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
index 7abdfaf..86d5769 100755
--- a/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
+++ b/ComicReader/src/com/blogspot/applications4android/comicreader/comics/CtrlAltDel.java
@@ -104,9 +104,9 @@ public class CtrlAltDel extends DailyComic {
                 i += 5;
               }
               else {
-                i = url.lastIndexOf ("cad-");
+                i = url.lastIndexOf ("Strip");
                 if (i >= 0) {
-                  i += 4;
+                  i += 5;
                 }
                 else {
                   return null;
@@ -308,15 +308,18 @@ public class CtrlAltDel extends DailyComic {
 
         protected SortedMap<Integer, CADStrip> getUrlsForMonth (String month)
         {
+            Log.d ("zzz", "getUrlsForMonth1 " + month);
             getMonths();
             TreeMap<Integer, CADStrip> l = mMonths.get (month);
             try {
               if (l == null) {
                 if (mMonths.containsKey (month)) {
+                  Log.d ("zzz", "getUrlsForMonth2");
                   l = new TreeMap<Integer, CADStrip>();
                   URI uri = new URI("https://cad-comic.com/wp-admin/admin-ajax.php");
                   String data = "action=custom_cat_search&post_cat=all&post_month="+month;
                   String sdata = Downloader.downloadToString(uri, data);
+                  Log.d ("zzz", "getUrlsForMonth3 " + sdata);
                   JSONObject root = new JSONObject(sdata);
                   JSONArray posts = root.getJSONArray("posts");
                   List<String> ret = new ArrayList<String>();
@@ -325,6 +328,7 @@ public class CtrlAltDel extends DailyComic {
                     String url = o.getString("comic");
                     if (url.lastIndexOf("KDM") >= 0) continue;
                     Calendar c = getTimeFromUrl (url);
+                    Log.d ("zzz", "getUrlsForMonth4 " + url + " " + c);
                     if (!month.equals (monthKey (c))) {
                       continue;
                     }
